dev/TeachingTip/TeachingTip_rs2_themeresources.xaml

```diff
@@ -4,13 +4,13 @@
     xmlns:local="using:Microsoft.UI.Xaml.Controls">
     <ResourceDictionary.ThemeDictionaries>
```

Comment:
You don't need to create a rs2_themeresource.
I guess you just want to make the one test happy. add UseNonstandardCondtionalXAML tag to page will make test happy.

eg:
    <Page Include="$(MSBuildThisFileDirectory)NavigationView_rs1_themeresources.xaml">
      <Version>RS1</Version>
      <Type>ThemeResources</Type>
      <UseNonstandardCondtionalXAML>true</UseNonstandardCondtionalXAML>
    </Page>

Reply:
Thanks, that's good feedback. I agree but I'd like to separate that cleanup from this change. Also trying to merge it made ThemeResourcesTests unhappy and we don't have an easy fix for that. I want to separate the two problems.